Topic-3-Javascript-DOM/2/styles.css

```diff
@@ -0,0 +1,38 @@
+section.hidden {
```

Comment:
Although the animation code is fine, you aren't ensuring that this runs after the document finish loading.